syft/pkg/cataloger/golang/internal/xcoff/file.go

```diff
@@ -0,0 +1,688 @@
+// Copyright 2018 The Go Authors. All rights reserved.
```

Comment:
nit: mind adding a comment at the top of source files copied in from the stdlib that they came from there? I think maybe even adding a new readme under `golang/internal` that explains context here would be great

Reply:
done